### PR TITLE
FR-42703 Park GeMS guiders when not in use.

### PR DESCRIPTION
**Pull request by jluhrs**
Seqexec had troubles with GeMS sequences where the old Tcl Seqexec had not. The problem was that TCS declared that the ODGW, and sometimes the Canopus guiders, where not in position, causing timeouts when applying offsets. The main difference between both Seqexec is that the Tcl Seqexec sends the park command for each Canopus guider and ODGW that is not in use.
I implemented the parks in this Seqexec hoping that that fixes the problem.

### CODE REVIEW

File: modules/server/src/main/java/edu/gemini/seqexec/server/tcs/ParkState.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+ * For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package edu.gemini.seqexec.server.tcs;
+
+public enum ParkState {
```
Comment (cquiroz) [inline modules/server/src/main/java/edu/gemini/seqexec/server/tcs/ParkState.java:8]: Why are these in Java? Due to the XML parser?

**Review by cquiroz — APPROVED** [review]
LGTM

File: modules/server/src/main/scala/seqexec/server/tcs/TcsSouthControllerEpicsAo.scala [hunk 2]
```diff
@@ -213,27 +207,34 @@ object TcsSouthControllerEpicsAo {
...
-        .flatMap(setCwfs1Guide(_, subsystems, current.cwfs1.tracking, demand.cwfs1.tracking)),
-      current.mapping
-        .get(Cwfs2)
-        .flatMap(setCwfs2Guide(_, subsystems, current.cwfs2.tracking, demand.cwfs2.tracking)),
-      current.mapping
-        .get(Cwfs3)
-        .flatMap(setCwfs3Guide(_, subsystems, current.cwfs3.tracking, demand.cwfs3.tracking)),
-      current.mapping
-        .get(Odgw1)
-        .flatMap(setOdgw1Probe(_)(subsystems, current.odgw1.tracking, demand.odgw1.tracking)),
```
Comment (jluhrs) [inline modules/server/src/main/scala/seqexec/server/tcs/TcsSouthControllerEpicsAo.scala:227]: The parking was implemented for ODGW, but this logic implied that the ODWG would be configured only if it was included in the Virtual Telescope Mapping, which never happened if the telescope configuration required it to be parked!

File: modules/server/src/main/java/edu/gemini/seqexec/server/tcs/ParkState.java (hunk 1 above)
Comment (jluhrs) [reply modules/server/src/main/java/edu/gemini/seqexec/server/tcs/ParkState.java:8]: Due to the giapi epics library.

**Pull request merged**